.gitignore

```diff
@@ -11,3 +11,5 @@ vendor
 .bundle
 .ruby-version
 .sass-cache
+.iml
+.idea
```

Comment:
In the future, I would make the changes to gitignore and updates two separate commits.